ui/components/Layout.tsx

```diff
@@ -151,25 +162,27 @@ function Layout({ className, children }: Props) {
         </TopToolBar>
         <Main wide>
           <NavContainer>
-            <NavContent>
-              <Tabs
-                centered={false}
-                orientation="vertical"
-                value={getParentNavValue(currentPage)}
-              >
-                {_.map(navItems, (n) => (
-                  <StyleLinkTab
-                    key={n.label}
-                    label={n.label}
-                    to={formatURL(n.value)}
-                    value={n.value}
-                    className={n.sub && "sub-item"}
-                    href={n.href}
-                    newTab={n.newTab}
-                  />
-                ))}
-              </Tabs>
-            </NavContent>
+            <FixedNav>
+              <NavContent>
```

Comment:
(for learning) why can't we merge these together? `FixedNav` is a simple `div` with added style. Can it be a added to `NavContent` and reduce the number of HTML nodes we generate?

Reply:
Because FixedNav is set to “position: fixed”, it will not take up any space. The NavContent div “pushes” the main content section right, giving us the layout we want.

Reply:
I understand that part of the CSS, but can't be NavContent `fixed` instead?

```
<FixedNav div>
  <NavContent div with pad>
    ...
  </div>
</div>
```
and just add this:
```
  background-color: white;
  position: fixed;
  height: 100%;
  top: 80;
  width: 240px;
  left: 0;
```

to `const NavContent = styled.div`. As I see that div has only 2 padding rule and bunch of definitions for child nodes.

It does not take up extra space on the UI, but in the DOM tree it does.

Reply:
`position: fixed` means that it won't take up space in the DOM. You will end up with `NavContent` on top of the main body content. It will look like this:
![Screenshot from 2022-03-08 07-31-44](https://user-images.githubusercontent.com/2802257/157270465-2d324bfb-f16d-470b-940b-7fc008d581b2.png)

Reply:
Ah, thanks.